MuteToggler@jebeaudet.com/files/MuteToggler@jebeaudet.com/applet.js

```diff
@@ -192,12 +213,51 @@ MyApplet.prototype = {
         }
     },
 
+    evaluate_soundcard: function() {
+      // only use specific soundcard if searchstring is not empty
+      if (this.soundcard.trim() !== '')
```

Comment:
Use `if (this.soundcard.trim().length > 0)` instead of `if this.soundcard.trim() !== '')`